src/CommunityToolkit.Maui/Views/WrapperControl.windows.cs

```diff
@@ -47,8 +46,8 @@ public void CleanUp()
 
 	protected override global::Windows.Foundation.Size ArrangeOverride(global::Windows.Foundation.Size finalSize)
 	{
-		view.IsInNativeLayout = true;
-		view.Frame = new Rectangle(0, 0, finalSize.Width, finalSize.Height);
+		//view.IsInNativeLayout = true;
```

Comment:
Can we delete this? I don't see an equivalent to the `IsInNativeLayout` property anymore 

```suggestion
```